libs/core/src/main/java/org/opensearch/core/common/xcontent/XContentParserUtils.java

```diff
@@ -30,10 +30,10 @@
  * GitHub history for details.
  */
 
-package org.opensearch.common.xcontent;
+package org.opensearch.core.common.xcontent;
```

Comment:
+1
It was placed here because the previous PR (the rote file move) needed to retain the namespace and this PR just moved to the core namespace. I think it makes sense to move to the core.xcontent package so I made the change.